src/test/resources/csv/parser-fixtures/hathaway_batch2_works_no_item_seq.csv

```diff
@@ -0,0 +1,20 @@
+Hathaway Manuscripts,ark:/21198/zz000bjfhp,ark:/21198/zz0009gsq9,Work,,text,Manuscripts,pd,"UCLA Charles E. Young Research Library Department of Special Collections, A1713 Young Research Library, Box 951575, Los Angeles, CA 90095-1575. E-mail: spec-coll@library.ucla.edu. Phone: (310)825-4991",ger|~|lat,"University of California, Los Angeles. Library. Performing Arts Special Collections",Hathaway 7,170/ 768 v.1|~|Hathaway collection of music manuscript leaves,"Breviary: Readings and Responsories.|~|Written in the Alemannic area, presumably in Switzerland.|~|Parchment (rather coarse), 13 long lines; ruling not discernible; pricking in the outer margin.  Written in a late, crude Caroline minuscule with squarish Alemannic features; dark brown ink.  Plain initials in red or stroked with red; initials in the text slashed with yellow; rubrics in red majuscule; neumes of St. Gall type.|~|Probably used as a flyleaf: notes in a German cursive hand (15th century) on both sides of both folios.|~|ff. 1, 2 and 3 from the same manuscript.  ff. 2-3 are consecutive.",Switzerland ?,205 x 145 mm,11th century,1001/1100,,,failed,
+Hathaway Manuscripts,ark:/21198/zz000bjfj6,ark:/21198/zz0009gsq9,Work,,text,Manuscripts,pd,"UCLA Charles E. Young Research Library Department of Special Collections, A1713 Young Research Library, Box 951575, Los Angeles, CA 90095-1575. E-mail: spec-coll@library.ucla.edu. Phone: (310)825-4992",lat,"University of California, Los Angeles. Library. Performing Arts Special Collections",Hathaway 8,170/ 768 v.1|~|Hathaway collection of music manuscript leaves,"Breviary.|~|Parchment, 1 leaf cropped at the top and at the margins.  30 long lines survive; ruling in lead.  Developed Caroline miniscule script; brown ink.  Plain capital initials in red; rubrics in red minuscule; neumes of St. Gall type.|~|Used as pastedown in binding; the verso is rubbed and stained.",Germany,203 x 130 mm,12th century,1100/1199,,,failed,
+"",,,,,,,,,,,,,,,,,,,,,
+"",,,,,,,,,,,,,,,,,,,,,
```

Comment:
Does this file need this many empty rows?

Reply:
No, I can remove